lrauv_ignition_plugins/test/test_spawn.cc

```diff
@@ -138,20 +147,20 @@ TEST(SpawnTest, Spawn)
   // Check vehicle positions
   EXPECT_NEAR(0.0, poses1.back().Pos().X(), 1e-3);
   EXPECT_NEAR(0.0, poses1.back().Pos().Y(), 1e-3);
-  EXPECT_NEAR(0.0, poses1.back().Pos().Z(), 1e-3);
+  EXPECT_NEAR(0.0, poses1.back().Pos().Z(), 1e-2);
   EXPECT_NEAR(0.0, poses1.back().Rot().Roll(), 1e-3);
-  EXPECT_NEAR(0.0, poses1.back().Rot().Pitch(), 1e-3);
+  EXPECT_NEAR(0.0, poses1.back().Rot().Pitch(), 1e-2);
   EXPECT_NEAR(0.0, poses1.back().Rot().Yaw(), 1e-3);
 
   EXPECT_NEAR(20.0, latLon1.back().X(), 1e-3);
   EXPECT_NEAR(20.0, latLon1.back().Y(), 1e-3);
-  EXPECT_NEAR(0.0, latLon1.back().Z(), 1e-3);
+  EXPECT_NEAR(0.0, latLon1.back().Z(), 1e-2);
 
   EXPECT_LT(100.0, poses2.back().Pos().X());
```

Comment:
Why are these lines checking less than rather than a specific value?

Reply:
I thought it wasn't necessary to go through the trouble of calculating the exact pose and instead just tested that "higher latitude and longitude is further North and East" #lazy. Doing the full check now: 067cfd9